README.md

```diff
@@ -30,43 +30,44 @@ All in all, RNFirebase provides much faster performance (~2x) over the web SDK a
 
 > '**?**' indicates partial support
 
-| Firebase Features      | v1.x.x  | v2.x.x  | v3.x.x | v3.1.x | v3.2.x | Web SDK |
-| ---------------------- | :---: | :---: | :---: | :---: | :---: | :---: |
-| **AdMob**                  | ❌ | ✅ | ✅ | ✅ | ✅ | ❌ |
-| **Analytics**              | ✅ | ✅ | ✅ | ✅ | ✅ | ❌ |
-| **App Indexing**           | ❌ | ❌ | ❌ | ❌ | ❌ | ❌ |
-| **Authentication**         | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ |
-| _-- Phone Auth_            | ❌ | ❌ | ✅ | ✅ | ✅ | ❌ |
-| **Core**                   | ❌ |**?**| ✅ | ✅ | ✅ | ✅ |
-|  _-- Multiple Apps_        | ❌ | ❌ | ✅ | ✅ | ✅ | ✅ |
-| **Cloud Firestore**        | ❌ | ❌ | ✅ | ✅ | ✅ |**?**|
-| **Cloud Messaging (FCM)**  | ✅ | ✅ | ✅ | ✅ | ✅ |**?**|
-| **Crashlytics**            | ❌ | ❌ | ❌ | ❌ | ✅ | ❌ |
-| **Crash Reporting**        | ✅ | ✅ | ✅ | ✅ | ✅ | ❌ |
-| **Dynamic Links**          | ❌ | ❌ | ❌ | ✅ | ✅ | ❌ |
-| **Invites**                | ❌ | ❌ | ❌ |**?**|**?**| ❌ |
-| **Performance Monitoring** | ✅ | ✅ | ✅ | ✅ | ✅ | ❌ |
-| **Realtime Database**      | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ |
-| _-- Offline Persistence_   | ✅ | ✅ | ✅ | ✅ | ✅ |**?**|
-| _-- Transactions_          | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ |
-| **Remote Config**          | ✅ | ✅ | ✅ | ✅ | ✅ | ❌ |
-| **Storage**                | ✅ | ✅ | ✅ | ✅ | ✅ |**?**|
+| Firebase Features          | v1.x.x | v2.x.x | v3.x.x | v3.1.x | v3.2.x | Web SDK |
+| -------------------------- | :----: | :----: | :----: | :----: | :----: | :-----: |
+| **AdMob**                  |   ❌   |   ✅   |   ✅   |   ✅   |   ✅   |   ❌    |
+| **Analytics**              |   ✅   |   ✅   |   ✅   |   ✅   |   ✅   |   ❌    |
+| **App Indexing**           |   ❌   |   ❌   |   ❌   |   ❌   |   ❌   |   ❌    |
+| **Authentication**         |   ✅   |   ✅   |   ✅   |   ✅   |   ✅   |   ✅    |
+| _-- Phone Auth_            |   ❌   |   ❌   |   ✅   |   ✅   |   ✅   |   ❌    |
+| **Core**                   |   ❌   | **?**  |   ✅   |   ✅   |   ✅   |   ✅    |
+| _-- Multiple Apps_         |   ❌   |   ❌   |   ✅   |   ✅   |   ✅   |   ✅    |
+| **Cloud Firestore**        |   ❌   |   ❌   |   ✅   |   ✅   |   ✅   |  **?**  |
+| **Cloud Messaging (FCM)**  |   ✅   |   ✅   |   ✅   |   ✅   |   ✅   |  **?**  |
+| **Crashlytics**            |   ❌   |   ❌   |   ❌   |   ❌   |   ✅   |   ❌    |
+| **Crash Reporting**        |   ✅   |   ✅   |   ✅   |   ✅   |   ✅   |   ❌    |
+| **Dynamic Links**          |   ❌   |   ❌   |   ❌   |   ✅   |   ✅   |   ❌    |
+| **Invites**                |   ❌   |   ❌   |   ❌   | **?**  | **?**  |   ❌    |
+| **Performance Monitoring** |   ✅   |   ✅   |   ✅   |   ✅   |   ✅   |   ❌    |
+| **Realtime Database**      |   ✅   |   ✅   |   ✅   |   ✅   |   ✅   |   ✅    |
+| _-- Offline Persistence_   |   ✅   |   ✅   |   ✅   |   ✅   |   ✅   |  **?**  |
+| _-- Transactions_          |   ✅   |   ✅   |   ✅   |   ✅   |   ✅   |   ✅    |
+| **Remote Config**          |   ✅   |   ✅   |   ✅   |   ✅   |   ✅   |   ❌    |
+| **Storage**                |   ✅   |   ✅   |   ✅   |   ✅   |   ✅   |  **?**  |
```

Comment:
tables are always a good example on why we should use Prettier for Markdown files ;)